resources/services/bigquery/bigquery_dataset_tables.go

```diff
@@ -20,7 +20,6 @@ func BigqueryDatasetTables() *schema.Table {
 		Description: "Model options used for the first training run These options are immutable for subsequent training runs Default values are used for any options not specified in the input query",
 		IgnoreError: client.IgnoreErrorHandler,
 		Resolver:    listBigqueryDatasetTables,
-		Options:     schema.TableCreationOptions{PrimaryKeys: []string{"dataset_cq_id", "id"}},
```

Comment:
Isn't that a root table?

Reply:
No its not.
https://github.com/cloudquery/cq-provider-gcp/blob/fcc304090e74395b41ffb60f2bb6fe3118da3ffe/resources/services/bigquery/bigquery_datasets.go#L101-L102